scripts/convertBooks.ts

```diff
@@ -36,7 +36,7 @@ export async function convertBooks(
             console.warn(`Language ${lang} already used in another collection. Proceeding anyway.`);
         }
         usedLangs.add(lang);
-        const abbr = collection.collectionAbbreviation;
+        const abbr = (collection.collectionAbbreviation ?? collection.id).toLowerCase();
```

Comment:
Proskomma has strict rules on the `docSet` value that (as far as I know) collection id might not meet. This works as long as it's been tested against Proskomma.